modules/installation-supported-aws-machine-types.adoc

```diff
@@ -412,7 +408,7 @@ The following Amazon Web Services (AWS) instance types are supported with {produ
 |===
 ====
 
-ifndef::aws-restricted-upi,aws-govcloud,aws-secret,aws-china,openshift-origin[]
+ifndef::aws-govcloud,aws-secret,aws-china,openshift-origin[]
```

Comment:
aws alone missing? 

Shouldn't it be?

> ifndef::aws,aws-govcloud,aws-secret,aws-china,openshift-origin[]

Reply:
Not including the `aws` alone is done on purpose. So that the Arm machine types appear in all the other AWS assembly’s other than aws on secret, aws in government regions, and aws in china regions becuase arm is not supported on those platforms yet.